lms/djangoapps/grades/config/tests/test_models.py

```diff
@@ -47,3 +47,22 @@ def test_persistent_grades_feature_flags(self, global_flag, enabled_for_all_cour
                 PersistentGradesEnabledFlag.feature_enabled(self.course_id_2),
                 global_flag and enabled_for_all_courses
             )
+
+    def test_enable_disable_course_flag(self):
+        """
+        Ensures that the flag, once enabled for a course, can also be disabled.
+        """
+        with persistent_grades_feature_flags(
+            global_flag=False,
+            enabled_for_all_courses=False,
+            course_id=self.course_id_1,
+            enabled_for_course=True
+        ):
+            # Prior to TNL-5698, creating a second object would fail due to db constraints
```

Comment:
Add a `self.assertTrue` to verify that the flag is enabled for the course at this point.